.drone.yml

```diff
@@ -190,6 +190,20 @@ steps:
     depends_on:
       - rootfs
 
+  - name: test-race
+    image: autonomy/build-container:latest
+    pull: always
+    environment:
+      BUILDKIT_HOST: tcp://buildkitd.ci.svc:1234
+      BINDIR: /usr/local/bin
+    commands:
+      - make test-race
+    volumes:
+      - name: dockersock
+        path: /var/run/
+    depends_on:
+      - lint
```

Comment:
I think the depends_on will need to be changed to `rootfs` or `osd` and `proxyd` ( think those are the two containers the tests depend on )

Reply:
test-race can't run containerd because it doesn't have Talos rootfs, so in fact it's the same as doing `go test` locally on your development tree. 

I think it's not quite possible to mix those as race detector depends on glibc, and our code is all built with musl